pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -221,6 +245,29 @@ function Meta(meta)
   end
 
   zotero.url = zotero.url .. '&citationKeys='
+
+  if string.match(FORMAT, 'odt') and config.csl_style ~= '' then
+    meta.ZOTERO_PREF_1 = string.format(
```

Comment:
What does _PREF_1 mean? And I only see this being set, but never used. Same goes for _PREF_2

Reply:
> What does _PREF_1 mean? And I only see this being set, but never used. Same goes for _PREF_2

These go as part of `meta.xml`.  See attached compilation.txt.

https://github.com/zotero/zotero-libreoffice-integration/blob/abac20076ef011737f699656f64f7a5dd1733394/build/source/org/zotero/integration/ooo/comp/Document.java#L107

https://github.com/zotero/zotero-libreoffice-integration/blob/abac20076ef011737f699656f64f7a5dd1733394/build/source/org/zotero/integration/ooo/comp/Document.java#L438